tests/test_project/tests/utils.py

```diff
@@ -125,7 +126,10 @@ def _get_menu_backdrop(self):
         return self.web_driver.find_element(By.CSS_SELECTOR, '.menu-backdrop')
 
     def _get_simple_input_filter(self):
-        return self.web_driver.find_element(By.CSS_SELECTOR, 'input[name=shelf]')
+        self.open(reverse('admin:test_project_shelf_changelist'))
```

Comment:
Is this method used only once? If so, please move this code to the test method, this separation isn't necessary and makes the code less readable with no benefits.